packages/utilities/carbon.yml

```diff
@@ -2,7 +2,7 @@
 library:
   id: ibmdotcom-utilities
   name: IBM.com Utilities
-  description: IBM.com ES6 utility classes
+  description: Apply JavaScript ES6 utility classes that adhere to IBM's web standards.
```

Comment:
is this final? airtable says in progress

Reply:
yup, Matt finalized the description